### ISSUE TITLE
Reported IP/Hostname in Settings Menu Resets on Server Reboot

### ISSUE THREAD

**Issue by CRY3rn** [jfurrow/flood#612]
## Expected Behavior
Should retain set IP or hostname in field.

## Current Behavior
Field is reset to 0.0.0.0 on server reboot.

## Your Environment
Debian Stretch 9.3
rtorrent 0.9.6-2+b1
Flood v1.0.0-281-gebc8bb7

**Comment by noraj1337** [comment]
@CRY3rn Which field are you talking about? Can you provide a screenshot or steps to reproduce?

**Comment by CRY3rn** [comment]
![flood-settings-reported-ip-hostname](https://user-images.githubusercontent.com/6866165/35487966-8d497c2c-0447-11e8-8274-9ddc191674d2.png)

**Comment by CRY3rn** [comment]
## Steps to Reproduce
Change the Reported IP/Hostname
Refresh the page to confirm change
Reboot server
Reported IP/Hostname is now 0.0.0.0

**Comment by noraj1337** [comment]
@CRY3rn The field is cleared, but do you know if the real reported IP has been reset to random or if the previously set IP is still applied?

**Comment by noraj1337** [comment]
Field : https://github.com/jfurrow/flood/blob/aa095625ab8c2255e309b5466e026167a73d3b67/client/src/javascript/components/modals/settings-modal/ConnectivityTab.js#L83-L92

**Comment by CRY3rn** [comment]
It looks like changes made in Flood are not permanent and it defaults to the IP stated in the .rtorrent.rc config file (or 0.0.0.0 if not specified).

So I did a couple tests.

I set the IP in the .rtorrent.rc config directly.
I then changed the IP in the Flood interface and saved it.
Refreshing the page kept displaying the new IP correctly.
Rebooting the server restored the IP to the one specified in the config.

**Comment by SanPilot** [comment]
Flood doesn't store rtorrent settings like limits, connections, peers, etc. These are all based off of what is reported by rtorrent, which in turn gets them from rtorrent.rc file on start.

**Comment by CRY3rn** [comment]
I was under the impression that Flood is capable of modifying rtorrent settings through the scgi port provided.

**Comment by SanPilot** [comment]
Yes, Flood can set them, but it doesn't store them in its own DB.

**Comment by CRY3rn** [comment]
Does rtorrent push the setting changes to the config file?

**Comment by bbashy** [comment]
Even ruTorrent doesn't save this to the config? I think it's expected that settings wouldn't persist on restarting it. Set it right in .rtorrent.rc :)
